refractr/refractr.yml

```diff
@@ -467,262 +496,325 @@ refracts:
-- https://perf-html.io/: cleopatra.io
+  # bug 1459170
+- perf-html.io/: cleopatra.io
+  status: 302
 
-# bug 1461516
-- https://www.mozilla.org/: smartdogz.org
+  # bug 1461516
+- www.mozilla.org/: smartdogz.org
+  status: 302
 
-# bug 1169224, 1473015
-- https://www.mozilla.org/:
+  # bug 1169224, 1473015
+- www.mozilla.org/:
   - mozilla.org.uk
   - www.mozilla.org.uk
+  status: 302
 
-# bug 1485991
-- https://onlinexperiences.com/scripts/Server.nxp?LASCmd=L:0&AI=1&InitialDisplay=1&ClientBrowser=0&ShowKey=44908: air.mozilla.org
+  # bug 1485991
+  # FIXME: previous redirect is not valid
+  #- onlinexperiences.com/scripts/Server.nxp?LASCmd=L:0&AI=1&InitialDisplay=1&ClientBrowser=0&ShowKey=44908: air.mozilla.org
+- air.mozilla.org/: air.mozilla.org
+  status: 308
 
-# bug 1419948
-# TODO: verify this works as before
-- https://web.archive.org/web/20180106223649/https://webfwd.org/:
+  # bug 1419948
+  # TODO: verify this works as before
+- web.archive.org/web/20180106223649/https://webfwd.org/:
   - webfwd.org
-  - www.webfwd.org
+  #- www.webfwd.org FIXME: gives SSL Cert Error
   - webfwd.net
   - www.webfwd.net
+  status: 307
 
-# bug 1462137
-- https://hubs.mozilla.com/:
+  # bug 1462137
+- hubs.mozilla.com/:
   - hub.mozilla.com
   - hub.mozilla.org
 
-# bug 1293353
-- https://mozilla.service-now.com/: thehub.mozilla.com
+  # bug 1293353
+- mozilla.service-now.com/: thehub.mozilla.com
+  status: 302
 
-# bug 1492531
-- https://wiki.mozilla.org/Pastebin: pastebin.mozilla.org
+  # bug 1492531
+- wiki.mozilla.org/Pastebin/: pastebin.mozilla.org
 
-# bug 1492428
-- https://www.mozilla.org/en-GB/firefox/new/?utm_source=mozilla-eu&utm_medium=referral:
+  # bug 1492428
+- www.mozilla.org/en-GB/firefox/new/?utm_source=mozilla-eu&utm_medium=referral:
   - mozilla.eu
   - www.mozilla.eu
   - mozila.eu
   - www.mozila.eu
 
   # bug 1492428
-- https://www.mozilla.org/fr/firefox/new/?utm_source=mozilla-fr&utm_medium=referral:
+- www.mozilla.org/fr/firefox/new/?utm_source=mozilla-fr&utm_medium=referral:
   - mozilla.fr
   - www.mozilla.fr
 
-# bug 1463086
-- https://mozilla.org/nl/:
+  # bug 1463086
+- mozilla.org/nl/:
   - mozilla.nl
   - www.mozilla.nl
 
-# bug 1502902
-- https://events.mozilla.org/techspeakers/: techspeakers.mozilla.org
-
-  # bug 1522224
-- https://testpilot.firefox.com/: contribute.testpilot.firefox.com
+  # bug 1502902
+  #- events.mozilla.org/techspeakers/: techspeakers.mozilla.org
+- developer.mozilla.com/communities/mozilla-tech-speakers/: techspeakers.mozilla.org
 
   # bug 1533035
-- https://mozilla.github.io/geckoview/: geckoview.dev
+- mozilla.github.io/geckoview/:
+  - gv.dev
+  - geckoview.dev
+  status: 302
 
-- https://mozilla.github.io/geckoview/: gv.dev
+  # FIXME: do we need these or assume that nginx ingress will http->https?
+- developer.mozilla.com/: developer.mozilla.com
 
-- https://developer.mozilla.org/: developer.mozilla.com
+  # FIXME: do we need these or assume that nginx ingress will http->https?
```

Comment:
I think this would _NOT_ get the domain into our ingress however.  So I have the ability to have a single `scalar` value (note: you ran into this elsewhere in the python) that means this is a straight http->https redirect with status: 301.  Even more simple that the current simple ones we have.  I think the python is ready for that but I don't think the jsonschema is.